fxa-auth-db-server/index.js

```diff
@@ -89,6 +89,11 @@ function createServer(db) {
       return db.deleteEmail(req.params.id, req.params.email)
     })
   )
+  api.get('/account/emails/:email',
```

Comment:
api nit: this doesn't have anything to do with an "account" so the leading "/account" on this URL is dead weight; just "/email" or "/secondaryEmail" or similar might prevent future confusion.